cit3.0-web/.eslintrc.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
     "no-unused-vars": "warn",
     "no-plusplus": ["error", { allowForLoopAfterthoughts: true }],
     "prettier/prettier": ["error"],
+    "react/no-array-index-key": "off",
   },
```

Comment:
I added some suggestions that might make adding this rule unnecessary. Let me know what you think

```suggestion
    "prettier/prettier": ["error"],
  },
```

Reply:
Yeah, this rule doesn't quite make much sense to start with. Usually index would be unique, therfore satisifying keying. I will see how it turns out when I make the changes above.

Reply:
My understanding is that it's not just about having the key be unique, but also always having the key always correspond to the same item so that React can avoid unnecessary re-renders. The indices in an array can change if the array changes in some way, whereas something like object IDs from the database should be more consistent.